js/adslot.js

```diff
@@ -21,10 +21,12 @@ export class AdSlot extends React.Component {
     adSenseAttributes: PropTypes.object,
     targetingArguments: PropTypes.object,
     onSlotRender: PropTypes.func,
+    onSlotRegister: PropTypes.func,
     onSlotIsViewable: PropTypes.func,
     onSlotVisibilityChanged: PropTypes.func,
     shouldRefresh: PropTypes.func,
     slotId: PropTypes.string,
+    extraClasses: PropTypes.string,
```

Comment:
WDYT about sticking to className (would allow styled-components and, I think, emotion, to hook in)?

Reply:
Good call, updated